cypress/integration/common/steps.js

```diff
@@ -371,7 +371,7 @@ When("I log in with the following credentials:", table => {
 
 When("open the notification menu and click on the first item", () => {
   cy.get(".notifications-menu").invoke('show').click(); // "invoke('show')" because of the delay for show the menu
-  cy.get(".notification-mention-post")
+  cy.get(".notification-link-for-test")
```

Comment:
I think we don't want classes just for tests... can we find this some other way?
CSS selectors?

Reply:
If we want, we could setup a build stage for fullstack testing only? Which is almost like production but without
`[data-test-*]`.

Reply:
Solved with your help @mattwr18 🤗